data/data/openstack/masters/main.tf

```diff
@@ -55,4 +75,3 @@ resource "openstack_compute_instance_v2" "master_conf" {
     openshiftClusterID = var.cluster_id
   }
 }
-
```

Comment:
hmm, why the drop?

Reply:
I'm fine with drive-by removal of useless trailing empty lines :p

Reply:
Lol it was irritating me, and I mindlessly removed it

Reply:
cool. keep it as-is